packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageModule.kt

```diff
@@ -0,0 +1,34 @@
+package expo.modules.image
+
+import android.graphics.drawable.Drawable
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.engine.DiskCacheStrategy
+import com.bumptech.glide.load.model.GlideUrl
+import com.facebook.react.bridge.Promise
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactContextBaseJavaModule
+import com.facebook.react.bridge.ReactMethod
+import java.lang.Exception
+
+class ExpoImageModule(val context: ReactApplicationContext) : ReactContextBaseJavaModule(context) {
+  override fun getName() = "ExpoImageModule"
+
+  @ReactMethod
+  fun prefetch(url: String, promise: Promise) {
+    val glideUrl = GlideUrl(url)
+    var result : Drawable? = null
+    try {
+      result = Glide.with(context)
+        .load(glideUrl)
+        .diskCacheStrategy(DiskCacheStrategy.DATA)
+        .submit()
+        .get()
+    } catch (e: Exception) {
+      promise.reject("E_PREFETCH_FAILURE", "Could not prefetch image: ${e.message}", e)
```

Comment:
We tend to follow NodeJS error codes naming style (`ERR_<module>_<short_name>`, [examples](https://nodejs.org/dist/latest-v14.x/docs/api/errors.html#errors_node_js_error_codes)) and then give as much context as possible. In this example it would be of tremendous help to embed failing `url`, but maybe it's already embedded in the error message right? 🤔 
```suggestion
      promise.reject("ERR_IMAGE_PREFETCH_FAILURE", "Failed to prefetch the image: ${e.message}", e)
```

Reply:
Yes, `url` is already embedded in the error message.